main.py

```diff
@@ -280,6 +296,11 @@ async def loop(adb_instance: ADB, config: AluneConfig):
                 logger.info("Queue lock released, likely loading into game now.")
             case GameState.IN_GAME:
                 logger.info("App state is in game, looping decision making and waiting for the exit button.")
+                _random_delay = 0
+                # If the surrend option is enabled AND if a max random delay have been set, choose a random value..
+                if config.get_auto_surrend() and config.get_auto_surrend_random_delay() > 0:
+                    _random_delay = _random.randint(1, config.get_auto_surrend_random_delay())
+
```

Comment:
This could likely be handled by the actual `get_auto_surrend_random_delay` function where it returns either 0 or the rand int between 1 and the delay, which would help make it a bit more readable